context/app/static/js/components/entity-search/MetadataMenu/MetadataMenu.jsx

```diff
@@ -38,16 +39,28 @@ function NotebookMenuItem(props) {
   );
 }
 
-function MetadataMenu({ entityType }) {
+function MetadataMenu({ entityType, results }) {
   const lcPluralType = `${entityType.toLowerCase()}s`;
   const { selectedHits, createNotebook, closeMenu } = useMetadataMenu(lcPluralType);
   const menuID = 'metadata-menu';
 
   const { selectedRows } = useStore();
-  const errorMessage =
-    selectedRows.size > 10
-      ? `You have selected ${selectedRows.size} datasets. Workspaces currently only supports up to 10 datasets. Please unselect datasets.`
-      : null;
+  const selectedRowsArray = useDatasetsAccessLevel(selectedRows.size > 0 ? [...selectedRows] : []).datasets;
+
+  const containsProtectedDataset = selectedRowsArray.some((item) => {
+    return item._source.mapped_data_access_level === 'Protected'; //eslint-disable-line
```

Comment:
Let's try to avoid doing full `eslint-disables` and stick to scoped ones in order to avoid disabling more rules than we intend to. You can see what rule is being broken by mousing over the error in VS code; you can also click through on the link to see more info about the error and why it's being thrown:

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/90b3fe82-9f13-4e08-aae6-3119259aea46)

I also recommend using `eslint-disable-next-line` instead of the same-line one for readability.

```suggestion
    //eslint-disable-next-line no-underscore-dangle
    return item._source.mapped_data_access_level === 'Protected'; 
```

Reply:
With the new configuration of the API results, this issue has been resolved.